docs/sources/reference/stdlib/array.md

```diff
@@ -32,3 +32,26 @@ Elements within the list can be any type.
 > array.concat([[1, 2], [3, 4]], [[5, 6]])
 [[1, 2], [3, 4], [5, 6]]
 ```
+
+## array.combine_maps
+
+{{< docs/shared lookup="stability/experimental.md" source="alloy" version="<ALLOY_VERSION>" >}}
```

Comment:
I wanted to avoid having two almost identical shared paragraphs but I'm ok to have both if "feature" is too vague for "component". @clayton-cornell what do you think?